tools/mod/go.mod

```diff
@@ -12,9 +12,12 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v5 v5.6.0
 )
 
+require sigs.k8s.io/kind v0.27.0
```

Comment:
Should we just move it to the `required` block above?

Reply:
Use `go get` command instead of manually updating the go.mod file